src/parsec_client/operations/list_providers.md

```diff
@@ -36,6 +36,10 @@ The version triplet returned by this operation (`version_maj`, `version_min` and
 the implementation version of the specific Parsec provider. For the Core Provider, this version is
 the implementation version of the whole Parsec service.
 
+The `providers` vector returned is in order of provider priority: the highest priority providers
+come first. The core provider will always come last. The provider at position zero, if not the core
+provider, can be treated as default provider by the client.
```

Comment:
Do you think it's needed to mention here that the client needs to check supported opcodes too? "can be treated as default" sounds like "you don't need to check anything else". I guess for now it doesn't matter, it possibly will in the future!

Reply:
Sure, will add a nuance.